crates/biome_js_analyze/tests/specs/correctness/useExhaustiveDependencies/valid.js.snap

```diff
@@ -242,4 +248,56 @@ function MyComponent22() {
 
 ```
 
+# Diagnostics
+```
+valid.js:121:5 lint/correctness/useExhaustiveDependencies ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! This hook does not specify all of its dependencies: a
+    			  		.b
+  
+    119 │ // Specified dependency cover captures
+    120 │ function MyComponent8({ a }) {
+  > 121 │     useEffect(() => {
+        │     ^^^^^^^^^
+    122 │       console.log(
+    123 │ 				a
+  
+  i This dependency is not specified in the hook dependency list.
+  
+    121 │     useEffect(() => {
+    122 │       console.log(
+  > 123 │ 				a
+        │ 				^
+  > 124 │ 			  		.b
```

Comment:
I think this is an unintended side-effect of this PR. When `a` is listed as a dependency, `a.b` is implicitly covered as well.